Formula/reaver.rb

```diff
@@ -22,15 +22,8 @@ class Reaver < Formula
   uses_from_macos "libpcap"
   uses_from_macos "sqlite"
 
-  # Adds general support for macOS in reaver:
-  # https://code.google.com/archive/p/reaver-wps/issues/detail?id=245
-  patch do
-    url "https://gist.githubusercontent.com/syndicut/6134996/raw/16f1b4336c104375ff93a88858809eced53c1171/reaver-osx.diff"
-    sha256 "2a8f791df1f59747724e2645f060f49742a625686427877d9f0f21dc62f811a7"
-  end
-
   def install
-    man1.install "docs/reaver.1.gz"
+    man1.install "docs/reaver.1"
```

Comment:
Does `make install` still not install this?

Reply:
I am not sure

Reply:
Can you try?

Reply:
I can try it later

Reply:
It does not. `man reaver` functions only with this line present in formula

Reply:
Can you report this issue upstream then

Reply:
@SMillerDev as the problem doesn't make things worse here in brew (related to previous version), should not we just merge this PR without fixing things upstream?

Reply:
We should never patch things without notifying upstream. That's how you get things like https://www.debian.org/security/2008/dsa-1571

Reply:
The issue is known upstream https://github.com/t6x/reaver-wps-fork-t6x/issues/195
It's only about documentation and I hardly imagine there's a chance we get CVE by including user manual to the package. On the other hand, having outdated package and postponing the merge _can_ bring CVE to end user.

Reply:
If the issue is known upstream, can you add a comment saying `# reported upstream in https://github.com/t6x/reaver-wps-fork-t6x/issues/195` so we know where the issue is being tracked?

Reply:
Done